templates/api/v1/kpi_setup/index.html.twig

```diff
@@ -0,0 +1,20 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Hello KpiSetupController!{% endblock %}
+
+{% block body %}
+<style>
+    .example-wrapper { margin: 1em auto; max-width: 800px; width: 95%; font: 18px/1.5 sans-serif; }
+    .example-wrapper code { background: #F5F5F5; padding: 2px 6px; }
+</style>
+
+<div class="example-wrapper">
+    <h1>Hello {{ controller_name }}! ✅</h1>
+
+    This friendly message is coming from:
+    <ul>
+        <li>Your controller at <code><a href="{{ '/var/www/html/RONY-JOBYER/PROJECTS/kpi-management-system/src/Controller/Api/v1/KpiSetupController.php'|file_link(0) }}">src/Controller/Api/v1/KpiSetupController.php</a></code></li>
+        <li>Your template at <code><a href="{{ '/var/www/html/RONY-JOBYER/PROJECTS/kpi-management-system/templates/api/v1/kpi_setup/index.html.twig'|file_link(0) }}">templates/api/v1/kpi_setup/index.html.twig</a></code></li>
```

Comment:
Still I am not clear about this file too. Moreover, plz use twig templating methods for href instead of direct folder name. It can be different in all other places.

Reply:
Done